src/utils/selectionItem.ts

```diff
@@ -1,9 +1,9 @@
 import { map } from "lodash";
-import { QuickPickItem } from "vscode"; // NOSONAR
+import { QuickPickItem, Uri } from "vscode";
 
 export class SelectionItem implements QuickPickItem {
-  public static async getSelectionItems(filesPaths: any): Promise<any> {
-    return map(filesPaths, uri => {
+  public static getSelectionItems(filesPaths: Uri[]): SelectionItem[] {
```

Comment:
You use this method in `mtaBuildCommand.ts` and `mtaDeployCommand.ts` but you didn't change the calls there (they still use `await`)

Reply:
done